tf2_eigen/package.xml

```diff
@@ -8,13 +8,13 @@
   <license>BSD</license>
 
   <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>cmake_modules</build_depend>
```

Comment:
Can you explain this a bit further so I understand?  From what I can tell, tf2_eigen.h includes:

#include <tf2/convert.h>
#include <Eigen/Geometry>
#include <geometry_msgs/PointStamped.h>
#include <geometry_msgs/PoseStamped.h>

So all of those would need build_export_depend (as you point out below).  Why would we also need to build_export_depend cmake_modules?